lib/core-generators/generator/templates/package.json.template

```diff
@@ -18,7 +18,7 @@
   "author": "<%= author %>",
   "license": "<%= typeof license !== 'undefined' ? license : '' %>",
   "dependencies": {
-    "lodash": "<%= typeof lodashSVR !== 'undefined' ? lodashSVR : '3.10.1' %>"
+    "@sailshq/lodash": "<%= typeof lodashSVR !== 'undefined' ? lodashSVR : '^3.10.3' %>"
```

Comment:
@rachaelshaw would you please try generating a generator, then running it?  That way we're 100% sure this works.  Also please review comments and requires in generated generator

Reply:
Tested running the generator, worked fine. Also double-checked the requires in the generated generator, it's all pointing to @sailshq/lodash.